source/runner/runners/_tests/vm2.test.ts

```diff
@@ -1,338 +0,0 @@
-        const results = await exec.runner.runDangerfileEnvironment(
-          [resolve(fixtures, "__DangerfileThrows.js")],
-          undefined,
-          runtime
-        )
-
-        expect(results.fails[0].message).toContain("Danger failed to run")
-        expect(results.markdowns[0].message).toContain("Error: failure")
-      })
-
-      it("handles running default export code", async () => {
-        const context = await setupDangerfileContext()
-        const runtime = await exec.runner.createDangerfileRuntimeEnvironment(context)
-        const results = await exec.runner.runDangerfileEnvironment(
-          [resolve(fixtures, "__DangerfileDefaultExport.js")],
-          undefined,
-          runtime
-        )
-        expect(results).toEqual({
-          fails: [],
-          messages: [],
-          markdowns: [],
-          warnings: [{ message: "Synchronous Warning" }],
-        })
-      })
-
-      it("handles running default export async code", async () => {
-        const context = await setupDangerfileContext()
-        const runtime = await exec.runner.createDangerfileRuntimeEnvironment(context)
-        const results = await exec.runner.runDangerfileEnvironment(
-          [resolve(fixtures, "__DangerfileDefaultExportAsync.js")],
-          undefined,
-          runtime
-        )
-        expect(results).toEqual({
-          fails: [],
-          messages: [],
-          markdowns: [],
-          warnings: [{ message: "Asynchronous Warning" }],
-        })
-      })
-
-      it("handles running multiple local files", async () => {
-        const context = await setupDangerfileContext()
-        const runtime = await exec.runner.createDangerfileRuntimeEnvironment(context)
-        const results = await exec.runner.runDangerfileEnvironment(
-          [resolve(fixtures, "__DangerfileTypeScript.ts"), resolve(fixtures, "__DangerfileAsync.ts")],
-          undefined,
-          runtime
-        )
-
-        expect(results).toEqual({
-          fails: [],
-          markdowns: [],
-          messages: [{ message: "Honey, we got Types" }],
-          warnings: [{ message: "Async Function" }, { message: "After Async Function" }],
-        })
-      })
-
-      it("handles running multiple dangerfiles with passed in content", async () => {
-        const context = await setupDangerfileContext()
-        const runtime = await exec.runner.createDangerfileRuntimeEnvironment(context)
-        const results = await exec.runner.runDangerfileEnvironment(
-          [resolve(fixtures, "__MadeUpDangerfileOne.ts"), resolve(fixtures, "__MadeUpDangerfileTwo.ts")],
-          ["markdown('hello')", "markdown('hello2')"],
-          runtime
-        )
-
-        expect(results).toEqual({
-          fails: [],
-          markdowns: [{ message: "hello" }, { message: "hello2" }],
-          messages: [],
-          warnings: [],
-        })
-      })
-    })
-  })
-})
```

Comment:
I can't reliably do the inline code evaluation inside Jest, so instead I powered up the fixtures and moved them to be on a push.